smart_contracts/contracts/test/dictionary-call/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "dictionary-call"
+version = "0.1.0"
+authors = ["Michał Papierski <michal@casperlabs.io>"]
+edition = "2018"
+
+[[bin]]
+name = "dictionary_call"
+path = "src/bin/main.rs"
+bench = false
+doctest = false
+test = false
+
+[features]
+std = ["casper-contract/std", "casper-types/std"]
+
+[dependencies]
+casper-contract = { path = "../../../contract" }
+casper-types = { path = "../../../../types" }
+dictionary = { path = "../dictionary" }
```

Comment:
Add trailing newline

Also:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

:wink:

Reply:
Fixed (and reconfigured my IDE)